appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -243,13 +243,40 @@ public boolean ShowFilterBar() {
    * @param itemsList a YailList containing the strings to be added to the ListView
    */
   @SimpleProperty(description="List of text elements to show in the ListView.  This will " +
-                "signal an error if the elements are not text strings.",
-      category = PropertyCategory.BEHAVIOR)
+          "signal an error if the elements are not text strings.",
+          category = PropertyCategory.BEHAVIOR)
   public void Elements(YailList itemsList) {
     items = ElementsUtil.elements(itemsList, "Listview");
     setAdapterData();
   }
 
+//  @SimpleProperty(description="Query results to show in the ListView. "
+//          "It will show the text values for the first variable found in the bindings.",
+//          category = PropertyCategory.BEHAVIOR)
+
+  /**
+   * Takes a set of query results, and displays values for the given variable name.",
+   * @param bindings
+   * @param showVariable
+   */
+  @SimpleFunction
+  public void LoadFromBindings(YailList bindings, String showVariable) {
+    this.bindings = bindings;
+    List<String> items = new ArrayList<>();
+    String varName = showVariable;
+    for(Object i : bindings.toArray()) {
+      YailDictionary binding = (YailDictionary) i;
+      if (varName == null)
+        varName = binding.keySet().iterator().next().toString();
+      if (!binding.containsKey(varName))
+        throw new YailRuntimeError("Variable name " + varName + " not found in bindings", "Error");
```

Comment:
Code style: Even for single-line bodies we surround the body with `{}`.

Reply:
Should be fixed now